Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.yml

```diff
@@ -0,0 +1,55 @@
+category: Messaging and Conferencing
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: AWS-SNS-Listener
+  version: -1
+configuration:
+- display: Long running instance
+  defaultvalue: 'true'
+  name: longRunning
+  type: 8
+  hidden: true
+  section: Connect
+  advanced: true
+  required: false
+- additionalinfo: "Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for AWS SNS Listener will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port."
+  display: Listen Port
+  name: longRunningPort
+  type: 0
+  required: true
+  section: Connect
+- display: Certificate (Required for HTTPS)
+  additionalinfo: "(For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use. (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported."
+  name: certificate
+  type: 12
+  section: Connect
+  required: false
+- display: Private Key (Required for HTTPS)
+  additionalinfo: "(For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use. (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported."
```

Comment:
This is the same description as for Certificate.  Should this be the Private key that the service should use?

Reply:
Thank you. Nice catch.